.gitignore

```diff
@@ -37,3 +37,6 @@ yarn-debug.log*
 
 # Ignore folder
 /vendor
+
+
+
```

Comment:
I could find any changed codes here but the project requires to:
- [ ] Add a comment to a post but currently am getting an error
![d](https://user-images.githubusercontent.com/44745855/148528915-51fed4cf-6918-40ce-bb9b-a7aa11b3f367.png)

- [ ] A user should be able to add a like to posts. please work on `likes` controller and it should have a create actions